0_From_Code_First_To_First_Principles/1_code_first/assignment/src/libs/api/ok.response.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ResponseDto } from "./response.dto.abstract";
+
+export class OkResponseDto<T> extends ResponseDto {
+  public readonly success = false;
```

Comment:
I got a `success: false` on the success response

Reply:
Nice catch! Corrected.